README.md

```diff
@@ -4,63 +4,75 @@
 - [GOV.UK Design System](https://design-system.service.gov.uk/) styles and components (via [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend))
 
-Components should be added to this gem if they are required in more than one application, otherwise they should be added to that application. All components should meet the same standards of browser and assistive technology support described in [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend#browser-and-assistive-technology-support).
+Only add a component to this gem if it is required in more than one application. Please do not add application-specific components to govuk_publishing_components.
+
+All components should meet the accessibility standards for browser and assistive technology support described in [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend#browser-and-assistive-technology-support).
 
 ## Technical documentation
 
-You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) or the local `startup.sh` script to run the app. Read the [guidance on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) to find out more about each approach, before you get started.
+### Run the application
+
+```sh
+./startup.sh
+```
 
-If you are using GOV.UK Docker, remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
+See also [Local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html).
 
-### Running the test suite
+### Run the test suite
 
 ```sh
+bundle install
+yarn install
 bundle exec rake
 ```
 
-To run JavaScript tests (only):
+### Run the JavaScript tests alone
+
+In browser:
 
 ```sh
-# run JS tests in browser
 yarn run jasmine:browser
+```
 
-# run JS tests on command line
+On command line:
+
+```sh
 yarn run jasmine:ci
 ```
 
-### Using the gem
+### Use the gem
 
 - [Install and use this gem](docs/install-and-use.md)
 - [Use a component in your application](docs/use-components.md)
 - [Test a branch of the gem](docs/test-a-branch-of-the-gem.md)
 
-### Managing the gem
+### Manage the gem
 
 - [Publish/release a new version of the gem](docs/publishing-to-rubygems.md)
 - [Keep this gem in sync with the Design System](docs/upgrade-govuk-frontend.md)
 - [Move a component from an application to the gem](docs/moving-components-upstream-into-this-gem.md)
 
-### Making components
+### Make a new component
 
 - [Generate a new component](docs/generate-a-new-component.md)
 - [Component conventions](docs/component_conventions.md)
 - [Component principles](docs/component_principles.md)
 
 ### Further documentation
 
-- [Testing a component](docs/testing-components.md)
-- [Component auditing](docs/auditing.md)
+- [Test a component](docs/testing-components.md)
+- [Audit a component](docs/auditing.md)
```

Comment:
Nitpick: `Audit a component` to me implies that this link will give information about how to audit a single component, or that there's a process for doing an audit of a component. But the component auditing gives information about where and how the components are used, which I think is slightly different.

Now that I write this I'm wondering if 'audit' is even the right word for this thing, but that's probably out of scope here! Otherwise all these changes look good 👍

Reply:
Ah cool, I'll leave 'Component auditing' as it was for now then. Thanks!